src/Components/3DTiles/TilesManager.js

```diff
@@ -227,6 +251,21 @@ export class TilesManager extends EventSender {
     }
   }
 
+  /**
+   * Sets the style of a particular tile.
+   * 
+   * @param {Int} tileId The tile
+   * identifier.
+   */
+  setDefaultStyleToTile(tileId) {
+    if(this.tiles[tileId]){
+      for(let i in this.tiles[tileId].cityObjects){
+        let meshId = this.tiles[tileId].cityObjects[i].meshId;
+        this.setStyle(this.tiles[tileId].cityObjects[i].cityObjectId, 'default' + tileId + 'm' + meshId);
```

Comment:
Create a method to construct the style id from a tileId and meshId instead of hard coding it may be better